src/test/java/org/opensearch/search/asynchronous/restIT/ApiParamsValidationIT.java

```diff
@@ -81,18 +81,6 @@ public void testSubmitDefaultWaitForCompletion() throws IOException {
         assertHitCount(submitResponse.getSearchResponse(), 5);
     }
 
-    /**
-     * run search on all indices
-     */
-    public void testSubmitSearchAllIndices() throws IOException {
```

Comment:
Any reason we don't support this?

Reply:
In the test set up we have index 6 documents. But when we do search on all indices in security enabled set up we are also searching the `.opendistro-security` index. Since we don't know how many docs will be present in that index this test is not deterministic. Hence removing it.